libs/application/template-api-modules/src/lib/modules/shared/shared.service.ts

```diff
@@ -264,4 +264,26 @@ export class SharedTemplateApiService {
     const fileContent = file.Body as Buffer
     return fileContent?.toString('base64') || ''
   }
+
+  async getAttachmentContentAsBlob(
+    application: ApplicationWithAttachments,
+    attachmentKey: string,
+  ): Promise<Blob> {
+    const fileName = (
+      application.attachments as {
+        [key: string]: string
+      }
+    )[attachmentKey]
+    const { bucket, key } = AmazonS3URI(fileName)
+    const uploadBucket = bucket
+    const file = await this.s3
+      .getObject({
+        Bucket: uploadBucket,
+        Key: key,
+      })
+      .promise()
+    const fileContent = file.Body as Buffer
+    const blob: Blob = new Blob([fileContent], { type: 'multipart/form-data' })
+    return blob
+  }
```

Comment:
Most of this can be moved to a separate function to reduce code duplication

```ts
  async getAttachmentContentAsBase64(
    application: ApplicationWithAttachments,
    attachmentKey: string,
  ): Promise<string> {
    const fileContent = await this.getAttachmentContentBuffer(
      application,
      attachmentKey,
    )
    return fileContent?.toString('base64') || ''
  }

  async getAttachmentContentAsBlob(
    application: ApplicationWithAttachments,
    attachmentKey: string,
  ): Promise<Blob> {
    const fileContent = await this.getAttachmentContentBuffer(
      application,
      attachmentKey,
    )
    const blob: Blob = new Blob([fileContent], { type: 'multipart/form-data' })
    return blob
  }

  private async getAttachmentContentBuffer(
    application: ApplicationWithAttachments,
    attachmentKey: string,
  ): Promise<Buffer> {
    const fileName = (
      application.attachments as {
        [key: string]: string
      }
    )[attachmentKey]
    const { bucket, key } = AmazonS3URI(fileName)
    const uploadBucket = bucket
    const file = await this.s3
      .getObject({
        Bucket: uploadBucket,
        Key: key,
      })
      .promise()
    return file.Body as Buffer
  }
```